Makefile

```
@@ -1,10 +1,4 @@
-.PHONY: help build build-server run start check-deps setup-certs install clean
-
-PODMAN := $(shell command -v podman 2>/dev/null)
-
-ifeq ($(PODMAN),)
-$(error Podman not found. Install: sudo dnf install podman podman-compose)
-endif
+.PHONY: help build build-server run start check-deps setup-certs install clean test test-race
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)